src/components/MarkerProps.tsx

```diff
@@ -120,7 +120,7 @@ export default interface MarkerProps {
   /**
    * Event fired with the user ends a drag for the annotation.
    */
-  onDragEnd?: () => void;
+  onDragEnd?: (newPosition: Coordinate) => void;
```

Comment:
Thank you for also updating the props accordingly. I forgot about this in my PR. However, I was wondering how you would implement additional data passing in the event. Like the API also has the data attribute, which I already included in my PR (although it will be implemented in another PR), and other attributes. We could pass them as additional parameters, though I personally would prefer passing over one object with both `{coordinates, data}` so if we have multiple things handed over we could avoid breaking changes and make it easier to implement without eslint unused argument warnings.

Reply:
The data passing feature is really not idiomatic in the context of a React API, so I don’t think that it is something we should support. In the React world, change events usually only pass the changed value so that it is easier to write event handlers (the component’s user already has access to all the other properties).

Reply:
Depends a lot on what events or libraries you are talking about, the rather "native" events do provide quite some context, while some events of libraries such as react-bootstrap provide rather only the value. In the end it comes down to preference, though we could definitely implement it like this and might add the `data` as a second argument later.